etc/taskcluster/decision_task.py

```diff
@@ -24,6 +24,9 @@ def main(task_for):
 
         # Implemented but disabled for now:
         linux_wpt = lambda: None  # Shadows the existing top-level function
+        # The magicleap build is broken until there's a surfman back end
+        magicleap_dev = lambda: None
+        magicleap_nightly = lambda: None
```

Comment:
These need to be added to mocked_only in this file.

Reply:
Done.